src/index.js

```diff
@@ -0,0 +1,200 @@
+"use strict";
+
+const state = {
```

Comment:
Love how you all chose to use this psuedostate! I think that it is great practice for what you will see in React though it will be utilized differently!